### PR TITLE
inject patches in `coda init`

### PR DESCRIPTION
**Pull request by huayang-codaio**
This PR allows `coda init` to patch `mold-source-map` using the one from packs-example (https://github.com/coda/packs-examples/pull/94). This avoids deprecation warnings from this zombie package.

### CODE REVIEW

File: cli/init.ts [hunk 1]
```diff
@@ -30,6 +39,9 @@ export async function handleInit() {
     .join(' ');
   spawnProcess(`npm install --save-dev ${devDependencyPackages}`);
 
+  // https://staging.coda.io/d/Quality-Tracker_d-GJF-DmEUK/Bugs_sucBW#Bugs_tuq-W/r20768&modal=true
```
Comment (patrick-codaio) [inline cli/init.ts:42]: Are you sure you want a link to staging in a public repo?

**Review by patrick-codaio — APPROVED** [review]

File: cli/init.ts (hunk 1 above)
Comment (huayang-codaio) [reply cli/init.ts:42]: ah, good catch. totally forgot this is a public repo..

**Review by jonathan-codaio — COMMENTED** [review]
I'm having a little trouble keep track of the similar PRs :) Is this one we still want to proceed with?

Seems fine to me.

**Comment by huayang-codaio** [comment]
yeah, this one allows `coda init` to let the developer pack repo to patch packages.

**Review by jonathan-codaio — APPROVED** [review]

**Comment by huayang-codaio** [comment]
ok so I have tested 
- node 16, 14.17.3 (which comes with npm@7+) and they worked. 
- node 14.18.2, 14.17.1, 14.13.1 (which comes with npm@6) and upgraded npm to @7. they worked as well. 

at the end of day, it's just a mess to try to auto create patches for developers.

**Comment by jonathan-codaio** [comment]
That makes sense. Thanks for trying!

**Pull request merged**